### ISSUE TITLE
Duplicate error messages for unused uses and not sorted use statements

### ISSUE THREAD

**Issue by ocean90** [slevomat/coding-standard#937]
This file produces correct output:

```php
<?php declare(strict_types = 1);

namespace Foo;

use const Foo\Bar\B;
use const Foo\Bar\A;

function something(): void {
	echo 'Hello World';
}
```

```
❯ phpcs namespace.php --standard=SlevomatCodingStandard

FILE: namespace.php
---------------------------------------------------------------------------------------------------
FOUND 5 ERRORS AFFECTING 2 LINES
---------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Type Foo\Bar\B is not used in this file.
 5 | ERROR | [ ] Type Foo\Bar\B should not be used, but referenced via a fully qualified name.
 6 | ERROR | [x] Use statements should be sorted alphabetically. The first wrong one is Foo\Bar\A.
 6 | ERROR | [x] Type Foo\Bar\A is not used in this file.
 6 | ERROR | [ ] Type Foo\Bar\A should not be used, but referenced via a fully qualified name.
---------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
---------------------------------------------------------------------------------------------------

Time: 61ms; Memory: 6MB
```

And this one not:

```php
<?php declare(strict_types = 1);

namespace Foo;

use const Foo\Bar\B;
use const Foo\Bar\A;

function something(): void {
	?>
	<span>
		<?php
		echo 'Hello World';
		?>
	</span>
	<?php
}
```

```
❯ phpcs namespace.php --standard=SlevomatCodingStandard

FILE: namespace.php
---------------------------------------------------------------------------------------------------
FOUND 11 ERRORS AFFECTING 2 LINES
---------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Type Foo\Bar\B is not used in this file.
 5 | ERROR | [ ] Type Foo\Bar\B should not be used, but referenced via a fully qualified name.
 5 | ERROR | [x] Type Foo\Bar\B is not used in this file.
 5 | ERROR | [x] Type Foo\Bar\B is not used in this file.
 6 | ERROR | [x] Use statements should be sorted alphabetically. The first wrong one is Foo\Bar\A.
 6 | ERROR | [x] Type Foo\Bar\A is not used in this file.
 6 | ERROR | [ ] Type Foo\Bar\A should not be used, but referenced via a fully qualified name.
 6 | ERROR | [x] Use statements should be sorted alphabetically. The first wrong one is Foo\Bar\A.
 6 | ERROR | [x] Type Foo\Bar\A is not used in this file.
 6 | ERROR | [x] Use statements should be sorted alphabetically. The first wrong one is Foo\Bar\A.
 6 | ERROR | [x] Type Foo\Bar\A is not used in this file.
---------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 9 MARKED SNIFF VIOLATIONS AUTOMATICALLY
---------------------------------------------------------------------------------------------------

Time: 68ms; Memory: 6MB
```

It seems like something gets confused with multiple opening/closing PHP tags.

Using a230516d742baedb2ff4b980ea90039dba36a6a5 (Should be unrelated to recent changes, was able to reproduce with 6.0.0 too)
PHP 7.3.16
PHP_CodeSniffer version 3.5.4

**Comment by kukulich** [comment]
Yes, we don’t have support for more PHP open tags in one file.

**Comment by kukulich** [comment]
However this can be probably fixed easy...

**Comment by kukulich** [comment]
@ocean90 Please try `dev-master`

**Comment by ocean90** [comment]
Tested 653364df41617768636176d6c68244cbcac773c7 and it's looking good for the unused use statements.

**Comment by kukulich** [comment]
Other duplicate errors fixed in master too.
